.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# who must review changes to this file
+/.github/CODEOWNERS @bluejekyll @djc
+
+# default owners for the whole repository
+* @bluejekyll @djc
+
+# Ferrous team can review changes to the E2E / integration tests
+/tests @japaric @justahero @listochkin @pvdrz
```

Comment:
Looks good.

Reply:
The last run took 3:34 minutes, but it can range from 2 to 5